projects/basisstatistiek.md

```diff
@@ -0,0 +1,49 @@
+---
+abstract: Descriptive statistics about the population and real estate of Amsterdam
+reusability: 1
+---
+
+# Basisstatistiek
+
+**This project in only available in Dutch**
```

Comment:
I wouldn't worry too much about this as I think the 1 already signifies this, so this could go under `## Reusability`.

Reply:
Die 1 staat er onder voorbehoud van onze discussie over taal als beperkende factor. Maar ik verplaats het wel even.